### PR TITLE
Add feature page: New policy language for MOM

### PR DESCRIPTION
**Pull request by akrejcir**
Changes proposed in this pull request:
- Add a feature page describing the design of a new policy language for MOM.

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @akrejcir

This pull request needs review by: @MarSik @doron-fediuck @aglitke

### CODE REVIEW

File: source/develop/release-management/features/sla/mom-declarative-language.html.md [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+- `max_relative_change` - Limits the relative change.
+
+```yaml
+rules:
+  balloon_shrink:
+    output: memory.balloon
+    min: policy.balloon_min
+    min_relative_change:
+        value: min_balloon_change_percent
+        interval: 10 sec
```
Comment (MarSik) [inline source/develop/release-management/features/sla/mom-declarative-language.html.md]: The minimal change does not need the `interval` property. It just won't be saved until the current difference is big enough.

The maximal allowed change needs the interval as it is part of the limit.

File: source/develop/release-management/features/sla/mom-declarative-language.html.md [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+
+The next section defines rules.
+
+```yaml
+rules:
+    cpu_limit:
+        output: cpu.max_load
+        min: 10
+        target: 0
+        function:
```
Comment (MarSik) [inline source/develop/release-management/features/sla/mom-declarative-language.html.md]: We should probably mention that in this case the same can be achieved using max_absolute_change: 1 / sec.

The function is supposed to decide on what the next value of the output variable is going to be. And we can even say that the default will be immediate jump to `target` (subject to change rate limits).

File: source/develop/release-management/features/sla/mom-declarative-language.html.md (hunk 1 above)
Comment (akrejcir) [reply source/develop/release-management/features/sla/mom-declarative-language.html.md]: Fixed.

File: source/develop/release-management/features/sla/mom-declarative-language.html.md (hunk 2 above)
Comment (akrejcir) [reply source/develop/release-management/features/sla/mom-declarative-language.html.md]: Done. I added a section listing possible functions and mentioning that 'constant' function is default.

**Comment by sandrobonazzola** [comment]
@MarSik can you please re-review?

**Pull request merged**